Hello Microverse PR

maov19 opened the pull request:
Pull request for Hello World excercise, day 2 of the program

maov19 committed:
Add node_modules to .gitignore file

maov19 committed:
Modified README file according to microverse standards

maov19 committed:
Add index and styles files

maov19 committed:
Modify .gitignore file

maov19 committed:
Add styleling correction file

maov19 committed:
Modified and added pending linters files

maov19 commented:
Pull request for Hello World excercise, day 2 of the program.
Modified for linters corrections.

maov19 committed:
Update README.md

deleted unnecessary comments from readme

OyePriscilla commented on README.md:
```diff
@@ -1,2 +1,143 @@
-Repo created for excercise "Set up a 'Hello Microverse' project" of Microverse program, day 2
+    <a name="readme-top"></a>
+    
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
- [x] This link is broken, if you don't have any logo at the moment kindly remove it so as to have a professional README file. ℹ️

maov19 replied:
The logo was removed, as requested.

OyePriscilla commented on README.md:
```diff
@@ -1,2 +1,143 @@
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
- [x] Kindly remove this template in order to have a clean file. 👍

OyePriscilla commented on README.md:
```diff
@@ -1,2 +1,143 @@
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> www.helloworld.com
+
+- [Live Demo Link](https://www.helloworld.com)
```
- [x]  Kindly remove this template in order to have a clean file. 👍
- [x] Please take note that the live demo should link to your page not an external page; if you don't have any live demo at the moment, you can just write "**The live link will be updated soon**". ℹ️

maov19 replied:
The template was removed and the text was replaced as requested.

OyePriscilla commented on README.md:
```diff
@@ -1,2 +1,143 @@
+> Tshepo Makama
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/festy1)
+
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/maov19)
+
+👤 **Author3**
+
+- GitHub: [@githubhandle](https://github.com/tshepo1)
```
- [x] You are the only author for the repo as of now I guess, kindly remove unnecessary details. You may also want to add your `twitter` and `email` links. 😃 This will really help the visitors to your repository. 👍

maov19 replied:
I don't have a twitter account! Maybe it's a good time to get one.
My e-mail info was added, as requested.

OyePriscilla commented on README.md:
```diff
@@ -1,2 +1,143 @@
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Goodbye World]**
+- [ ] **[Good Morning World]**
+- [ ] **[Good Night World]**
+
```
- [optional] I am not so sure if these are suppose to be future features, I am thinking feature futures can be` Implement mobile version`, `Implement desktop version`, `Make project dynamic` ... 👍

maov19 replied:
Feature were modified as requested.

OyePriscilla commented on README.md:
```diff
@@ -1,2 +1,143 @@
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
- [x] Please this link is broken, kindly create a LICENSE file in your root folder, and customize [this text](https://gist.github.com/adamveld12/b64e74d14d7d9043b85c). Obviously, you've made a great effort but you are almost there. 👍

maov19 replied:
LICENSE.md file was created, as requested.

OyePriscilla reviewed (changes requested):
Hi @maov19 😃, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close-so-close-seth-meyers](https://user-images.githubusercontent.com/103386183/196425615-8927e377-c543-407b-8da4-5c09fecc261b.gif)

## Highlights
Github flow observed✔️
Linters Set up correctly ✔️ 

## Required Changes ♻️

**- [ ] Kindly add a more descriptive summary template to your Pull Request.**

## Optional suggestions

**[optional] Please for your subsequent PRs, always ensure that your branch name describes the current milestone. 👍** 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

maov19 committed:
Update README.md

Remove initial logo
Remove templates in features and live link 
Modify authors info

maov19 committed:
Update README.md

removed wrong div at beggining

maov19 committed:
Create LICENSE.md

As requested in CR

maov19 commented:
Priscila, thanks for your feedback.
Changes were performed, as requested. Please let me know if there anything else I should work on.

brainconnect93 commented on index.html:
```diff
@@ -0,0 +1,13 @@
+    <title>Document</title>
+</head>
+<body>
+    <h1>"Hello Microverse!"</h1>    
```
- [Optional] Kindly give your `h1` tag a class name and use the name to style your content in the CSS file 👍 

E.G
```HTML
   <h1 class="header">"Hello Microverse!"</h1>
```

maov19 replied:
A header class was added to the h1 tag

brainconnect93 commented on styles.css:
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```
- [ ] [Optional] Kindly use the class name created in your `h1` ta to style it here

E.G

```CSS
.header {
  color: blue;
}
```

maov19 replied:
Modify h1 tag for ,header class

brainconnect93 commented on README.md:
```diff
@@ -1,2 +1,123 @@
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
- [ ] As requested by the previous reviewer, this MIT license is still broken kindly fix it. I know you made a great effort to get it fixed but it is still not working. You can achieve this by adding the `.md` extension name to the link.

E.g 

```
This project is [MIT](./LICENSE.md) licensed.
```

Bravo 👍 👍

maov19 replied:
Add LICENSE.md to license line

brainconnect93 reviewed (changes requested):
### Hi @maov19, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media.tenor.com/isiLRnxz3zwAAAAM/cat-driving-serious.gif" width="500px" height="300px">

### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

maov19 committed:
Update README.md

Modify line 121 by adding .md to the LICENSE file

maov19 committed:
Update index.html

Add class to h1

maov19 committed:
Update styles.css

Modified h1 for .header class

maov19 commented:
Hi! Thanks for your comments, everything should be solved by now.
Let me know if there is something else I should work on.

brainconnect93 reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @maov19, 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 


<img src="https://i.pinimg.com/originals/f8/3d/cf/f83dcf0c379de146ef962251952577ab.gif">

## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
